contrib/babelfishpg_tsql/src/pltsql_utils.c

```diff
@@ -2184,6 +2372,13 @@ char
 
 	initStringInfo(&query);
 
+	if(list_length(column_name) > 32)
+ 	{
+ 		ereport(ERROR,
+ 			(errcode(ERRCODE_INVALID_OBJECT_DEFINITION),
+ 				errmsg("Fulltext Index on more than 32 columns is currently not supported in Babelfish")));
```

Comment:
Let's align this message as we discussed: `"Full-text index creation with more than 32 columns is not currently supported in Babelfish."`